### DISCUSSION TITLE
Change the default/empty arc color from rgb(92,92,92)

### DISCUSSION THREAD

**Discussion by buildpath-ian** [antoniolago/react-gauge-component#26] [Q&A] [answered]
How can I modify the color of the dark grey default arc ?

**Comment by antoniolago** [comment]
Currently there's no implementation of this feature, but it's an interesting idea, created an issue from this. Thanks

**Comment by antoniolago** [comment, marked as answer]
Implemented a property to handle this on v1.2.3 (https://github.com/antoniolago/react-gauge-component/pull/56).

Please try arc.emptyColor property.
